openpgp/packet/public_key.go

```diff
@@ -571,7 +571,11 @@ func (pk *PublicKey) VerifySignature(signed hash.Hash, sig *Signature) (err erro
 	hashBytes := signed.Sum(nil)
 
 	if hashBytes[0] != sig.HashTag[0] || hashBytes[1] != sig.HashTag[1] {
-		return errors.SignatureError("hash tag doesn't match")
+		// NOTE(maxtaco) 2016-08-22
+		// Don't do anything in this case. Some GPGs generate bad
+		// 2-byte hash prefixes, but GPG also doesn't seem to care on
+		// import. See BrentMaxwell's key. I think it's safe to disable
+		// this check!
```

Comment:
Mild preference for just commenting out the whole thing in place, rather than leaving just part of the structure.